code/midtown/mmai/aiVehicleOpponent.cpp

```diff
@@ -73,3 +81,49 @@ void aiVehicleOpponent::Init(i32 opp_id, aiRaceData* race_data, char* race_name)
     }
     AudIndexNumber = -1;
 }
+
+void aiVehicleOpponent::Update()
+{
+    if (AudIndexNumber == -1)
+    {
+        AddToAiAudMgr();
+    }
+
+    Vector3 car_pos_diff = Car.Sim.ICS.Matrix.m3 - AIMAP.PlayerPos();
+    f32 car_dist_sqr = car_pos_diff ^ car_pos_diff;
+
+    if (IsSemi || (car_dist_sqr) < 40000.0f)
```

Comment:
Unecessary parenthesis around car_dist_sqr, also replace the `40000.0f` with `(200.0f*200.0f)`